src/styles/colors.scss

```diff
@@ -30,9 +30,6 @@
   --color-border-display: var(--color-neutral-3);
   --color-border-input: var(--color-neutral-4);
   --color-border-input-disabled: var(--color-neutral-6);
-
-  // Other Contextual Colors ////////////////
-  --bg-feedback-success: #cee8de;
```

Comment:
We could create a `--bg-success: var(--color-success)` below `--bg-selected-outline` (line 28)